stl/inc/format

```diff
@@ -246,7 +246,7 @@ public:
 };
 
 template <class _Visitor, class _Context>
-auto visit_format_arg(_Visitor&& _Vis, basic_format_arg<_Context> _Arg) {
+_NODISCARD auto visit_format_arg(_Visitor&& _Vis, basic_format_arg<_Context> _Arg) {

```

Comment:
I think `[[nodiscard]]` is inappropriate here. A user could pass a visitor with side effects and an ignorable return value, in which case discarding the return from `visit_format_arg` does not indicate a bug.

Reply:
I agree with this assessment.